lib/matplotlib/tests/test_quiver.py

```diff
@@ -277,3 +278,19 @@ def test_quiver_setuvc_numbers():
 
     q = ax.quiver(X, Y, U, V)
     q.set_UVC(0, 1)
+
+
+@check_figures_equal()
```

Comment:
Probably no need to check other types:
```suggestion
@check_figures_equal(extensions=['png'])
```